WcaOnRails/spec/controllers/delegate_reports_controller_spec.rb

```diff
@@ -54,7 +54,7 @@
       expect(comp.delegate_report.remarks).to eq "My new remarks"
     end
 
-    it "can edit report before comp is over, but cannot post report" do
+    it "can edit report before comp is over, and can post report" do
```

Comment:
I was thinking about when it was `!end_date.nil? && end_date < Date.today - 1.day` for a moment.

Outdated comment.